### ISSUE TITLE
Drop suppor of NS?

### ISSUE THREAD

**Issue by mattmega4** [PureLayout/PureLayout#185]
Just curious if there are any plans to drop NS...such as NSArray and use just Array?

**Comment by chrisballinger** [comment]
That doesn't really make sense because this is an Objective-C library. They will import into Swift as Array.

**Comment by pronebird** [comment]
@chrisballinger no they won't. `autoRemove` does not work on arrays until you case it to NSArray.

**Comment by chrisballinger** [comment]
@pronebird You're right, but you still can't make an Objective-C category on `Array` since it doesn't exist. I think someone could make a Swift compat shim that bridges the two via an `Array` extension.

**Comment by pronebird** [comment]
@chrisballinger you think it's possible to bundle that shim into PureLayout and have it working in Swift projects?
